packages/babel-core/src/transformation/file/options/build-config-chain.js

```diff
@@ -114,7 +115,16 @@ class ConfigChainBuilder {
       options = jsonCache[content] = jsonCache[content] || json.parse(content);
       if (key) options = options[key];
     } catch (err) {
-      err.message = `${loc}: Error while parsing JSON - ${err.message}`;
+      const code = codeFrame(content, err.lineNumber, err.columnNumber);
+
+      err.message = `
```

Comment:
Should we keep this more in line with the parser frame error in https://github.com/babel/babel/blob/fc9becfdf2a3dc212cf28a57071f33cdf2442698/packages/babel-core/src/transformation/normalize-file.js#L152 so the error message and filename are first, then the code frame?